maven-plugin/pom.xml

```diff
@@ -66,9 +66,8 @@
       <artifactId>maven-plugin-annotations</artifactId>
     </dependency>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <scope>test</scope>
+      <groupId>xml-apis</groupId>
+      <artifactId>xml-apis</artifactId>
```

Comment:
Where does this one come from? These APIs are built-in to the JDK. At the very least, this should probably be marked "provided", but it can get a [lot more complicated than that][1].

[1]: http://stackoverflow.com/questions/11677572/dealing-with-xerces-hell-in-java-maven